src/api/spec/queries/packages_finder_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
+
+RSpec.describe PackagesFinder, vcr: true do
+  describe '#by_package_and_project' do
+    let(:package_finder) { PackagesFinder.new }
```

Comment:
`package_finder` is not used in the test. the same happens in the others `describe`